english/interact/fork/follow/follow.md

```diff
@@ -4,4 +4,4 @@ With the fork still in hand, you pace yourself towards the hopping Mr marshmallo
 
 At one corner of the marshmallow room, you notice a large dark hole and marshmallow just jump straight into it!
 
-Do you:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.